tensorflow/core/kernels/mkl_matmul_op.cc

```diff
@@ -0,0 +1,157 @@
+/* Copyright 2015 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Thanks for the feedback. I haven't yet looked at batch Matmul, but, yes, we will add MKL support there too